es6-classes/circle.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-undef -- Shape is imported by .html */
+/* exported Circle */
+
+class Circle extends Shape {
+  constructor(radius) {
+    super(area, perimeter);
```

Comment:
`area` and `perimeter` haven't been defined. You should be passing in a number here, utilizing your `radius` parameter.